tests/TestCase/Core/Configure/Engine/JsonConfigTest.php

```diff
@@ -96,7 +96,7 @@ public function testReadWithNonExistentFile()
      * Test reading an empty file.
      *
      * @expectedException \Cake\Core\Exception\Exception
-     * @expcetedExceptionMessage Decoding JSON config file "empty.json" did not return any array
+     * @expectedExceptionMessage Decoding JSON config file "empty.json" did not return an array
```

Comment:
You could use a regexp for the common part of the error message `config file "empty.json"`